framework/src/problems/FEProblemBase.C

```diff
@@ -6297,10 +6332,6 @@ FEProblemBase::computeResidualAndJacobian(const NumericVector<Number> & soln,
     }
     catch (MooseException & e)
     {
-      _console << "\nA MooseException was raised during Auxiliary variable computation.\n"
-               << "The next solve will fail, the timestep will be reduced, and we will try again.\n"
-               << std::endl;
```

Comment:
It looks like we would...

Reply:
I agree that this is valuable information that we don't want to lose. I believe what I was thinking was that this would get printed out later, but in looking at the output from the test case that motivated this (which also triggered some of these), I'm not getting what I wanted. I'll look into this.

Reply:
I'm pretty sure this is how we want this. The part about the solution failing and the step getting cut is printed out in `FEProblemBase::checkExceptionAndStopSolve()`